src/SignalR/common/SignalR.Common/src/Protocol/SequenceMessage.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.SignalR.Protocol;
+
+/// <summary>
+/// Represents the restart of the sequence of messages being sent. <see cref="SequenceId"/> is the starting ID of messages being sent, which might be duplicate messages.
+/// </summary>
+public sealed class SequenceMessage : HubMessage
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="SequenceMessage"/> class.
+    /// </summary>
+    /// <param name="sequenceId">Specifies the ID messages being received from now on should start at.</param>
```

Comment:
Wording is a bit awkward. How about something like

```suggestion
    /// <param name="sequenceId">Specifies the starting ID for messages that will be received from this point onward.</param>
```